docs/dev/table/connect.md

```diff
@@ -621,7 +621,7 @@ The connector operates in [upsert mode](#update-modes) and exchanges UPSERT/DELE
     // optional: configure how to buffer elements before sending them in bulk to the cluster for efficiency
     .disableFlushOnCheckpoint()    // optional: disables flushing on checkpoint (see notes below!)
     .bulkFlushMaxActions(42)       // optional: maximum number of actions to buffer for each bulk request
-    .bulkFlushMaxSize(42)          // optional: maximum size of buffered actions (in MB) per bulk request
+    .bulkFlushMaxSize("42 mb")     // optional: maximum size of buffered actions (in MB) per bulk request
```

Comment:
It is no longer `(in MB)`, by default if no unit is provided it will be treated as bytes. I would also add info about the minimal restriction here.